server/index.js

```diff
@@ -18,6 +18,12 @@ nextApp.prepare().then(() => {
   /* APIs go there */
   expressServer.use('/api', api);
 
+  /* Clean URL for rendering Product Information*/
+  expressServer.get('/product/:id', (req, res) => {
+    const actualPage = '/product';
+    nextApp.render(req, res, actualPage, req.params);
```

Comment:
Instead of `actualPage` we can probably use `req.url` (verify if it's `/product` only)